src/graphql/user.ts

```diff
@@ -1,7 +1,39 @@
-import { extendType, inputObjectType, nonNull, objectType } from 'nexus';
+import { extendType, FieldResolver, inputObjectType, nonNull, objectType } from 'nexus';
 import { User } from '../entity/User';
 import { AppDataSource } from '../data-source';
 
+const handlePasswordValidation = (password: string) => {
+  const regex = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{6,}$/; //REGEX: At least 6 characters, one letter and one digit
+
+  return regex.test(password);
+};
+
+const handleCreateUserResolve: FieldResolver<'Mutation', 'createUser'> = async (_parent, args) => {
```

Comment:
os nomes de funções ficariam mais claros sem o `handle*` aqui
poderiam ser createUserResolver() e validatePassword() mesmo
no react tem o padrão em nomear com o handle* em alguns casos por conta de estar focado em tratar eventos mesmo (o click de um botao, por exemplo), aqui se for usar essa abordagem vai ficar com um monte de handle poluindo a leitura :thinking: